replication/README.md

```diff
@@ -312,3 +316,38 @@ message ResyncVolumeResponse{
 | Not authenticated                                | 16 UNAUTHENTICATED    | The invoked RPC does not carry secrets that are valid for authentication.                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | Caller SHALL either fix the secrets provided in the RPC, or otherwise regalvanize said secrets such that they will pass authentication by the Plugin for the attempted RPC, after which point the caller MAY retry the attempted RPC. |
 | Error is Unknown                                 | 2 UNKNOWN             | Indicates that a unknown error is generated                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | Caller MUST study the logs before retrying                                                                                                                                                                                            |
+
+### GetVolumeReplicationInfo
+
+```protobuf
+// GetVolumeReplicationInfoRequest holds the required information to get
+// the Volume replication info.
+message GetVolumeReplicationInfoRequest {
+  // The identifier for this volume, generated by the plugin during
+  // CreateVolume CSI RPC call.
+  // This field is REQUIRED.
+  // This field MUST contain enough information to uniquely identify
+  // this specific volume vs all other volumes supported by this plugin.
+  // This field SHALL be used by the CO in subsequent calls to refer to
+  // this volume.
+  string volume_id = 1;
+  // Plugin specific parameters passed in as opaque key-value pairs.
+  map<string, string> parameters = 2;
```

Comment:
why do we need parameters for Get call?

Reply:
Not sure, just added a basic template based on other calls, if not required can update later?

Reply:
Removing later can create inconsistency in the RPC pack and unpack. I would suggest removing it and adding it when required.